core/src/db/engine/ExecutionEngineImpl.cpp

```diff
@@ -380,9 +403,18 @@ ExecutionEngineImpl::Serialize() {
 
 Status
 ExecutionEngineImpl::Load(bool to_cache) {
-    index_ = std::static_pointer_cast<knowhere::VecIndex>(cache::CpuCacheMgr::GetInstance()->GetIndex(location_));
-    bool already_in_cache = (index_ != nullptr);
+#ifdef MILVUS_FPGA_VERSION
+    auto  cache_index__ = std::static_pointer_cast<knowhere::VecIndex>(cache::FpgaCacheMgr::GetInstance()->GetIndex(location_));
+    bool already_in_caches = (cache_index__ != nullptr);
+    if (already_in_caches) {
+      LOG_ENGINE_DEBUG_ << "jack fpga cache no need to load again" ;
```

Comment:
Don't use personal name in log or error message.
Sometimes we can use personal name in code comment such as "TODO: (xxx)", but for log and message it is not allowed.

Reply:
sorry for forget to delete it , i will modify.